jobs/build-arch.Jenkinsfile

```
@@ -306,6 +306,14 @@ lock(resource: "build-${params.STREAM}-${basearch}") {
             pipeutils.build_artifacts(pipecfg, params.STREAM, basearch)
         }
 
+        // secex specific tests
+        if (shwrapCapture("cosa meta --get-value images.qemu-secex") != "None") {
+            stage("Kola:TestSecex") {
+                kola(cosaDir: env.WORKSPACE, arch: basearch, skipBasicScenarios: true, skipSecureBoot: true, skipUpgrade: true,
```

Passing `extraArgs` already implies that we won't run basic scenarios, so we can simplify this to

```suggestion
                kola(cosaDir: env.WORKSPACE, arch: basearch, skipUpgrade: true,
```